File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-define-and-select-ranges-in-documents_12.cs
            Word.Document document = this.Application.ActiveDocument;

            if (document.Sentences.Count >= 2)
            {
                object startLocation = document.Sentences[2].Start;
                object endLocation = document.Sentences[2].End;

                // Supply a Start and End value for the Range. 
                rng = document.Range(ref startLocation, ref endLocation);

                // Select the Range.
                rng.Select();
            }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-define-and-select-ranges-in-documents_12.vb
        Dim document As Word.Document = Me.Application.ActiveDocument
        If document.Sentences.Count >= 2 Then

            Dim startLocation As Object = document.Sentences(2).Start
            Dim endLocation As Object = document.Sentences(2).End

            ' Supply a Start and End value for the Range.
            rng = document.Range(Start:=startLocation, End:=endLocation)

            ' Select the Range
            rng.Select()
        End If